src/__tests__/pagopa.test.ts

```diff
@@ -152,9 +155,9 @@ describe("QrCodeFromString", () => {
     const qrCodeSrts: ReadonlyArray<string> = [
       "XAGOPA|002|123456789012345678|12345678901|1234567801", // invalid identifier
       "PAGOPA|003|123456789012345678|12345678901|1234567801", // invalid version
-      "PAGOPA|003|123456789012345678|12345678901|123456780X", // invalid amount
-      "PAGOPA|003|12345678901234567X|12345678901|1234567800", // invalid paymentNumber
-      "PAGOPA|003|123456789012345675|X2345678901|1234567800" // invalid fiscal code
+      "PAGOPA|002|123456789012345678|12345678901|123456780X", // invalid amount
+      "PAGOPA|002|12345678901234567X|12345678901|1234567800", // invalid paymentNumber
+      "PAGOPA|002|123456789012345675|X2345678901|1234567800" // invalid fiscal code
```

Comment:
This PR should not be concerned with this -- but I noticed this and decided to fix it anyway. I'm expecting those values to be `002`s, not `003`s, right? Otherwise, the conversion may fail for the wrong reason

Reply:
that's right, good catch, thank you